statistics/handle/handle.go

```diff
@@ -1963,45 +1964,6 @@ func (h *Handle) GetPredicateColumns(tableID int64) ([]int64, error) {
 // Max column size is 6MB. Refer https://docs.pingcap.com/tidb/dev/tidb-limitations/#limitation-on-a-single-column
 const maxColumnSize = 6 << 20
 
-// RecordHistoricalStatsToStorage records the given table's stats data to mysql.stats_history
-func (h *Handle) RecordHistoricalStatsToStorage(dbName string, tableInfo *model.TableInfo) (uint64, error) {
```

Comment:
Why is this method changed? It seems like we don't need to move it.

Reply:
I changed it by mistake when resolving conflicts. Cancel the change now.